src/contracts/constants.ts

```diff
@@ -1,4 +1,4 @@
-import { getConfig } from 'config'
+import { getConfig } from '../config'
```

Comment:
I think it's not required now that we are using tsx but I'm happy with it on my side, `'config'` looks too confusing to me

Reply:
55a6e7975388837d7fefaa9ee89e00fef588c467